Packs/CybleEventsV2/Playbooks/playbook-Cyble_Vision_Alert_V2.yml

```diff
@@ -91,13 +124,13 @@ inputs:
     simple: "2023-05-30T00:00:00+00:00"
   required: true
   description: ""
-  playbookInputQuery: null
+  playbookInputQuery:
 - key: end_date
   value:
     simple: "2023-05-31T00:00:00+00:00"
```

Comment:
Hi @melamedbn,
Are you talking about these dates? If yes, then I am not sure what to put here?
Does this section have same inputs as in line 53 and 61 for start and end dates?

Reply:
Hi @melamedbn,
Can we please escalate this? Let me know the expected edit in above lines 124 & 130.